guides/ecto.md

```diff
@@ -543,6 +580,23 @@ $ mix ecto.migrate
 
 ## Other options
 
-While Phoenix uses the [Ecto project](https://hexdocs.pm/ecto) to interact with the data access layer, there are many other data access options, some even built into the Erlang standard library. [ETS](http://www.erlang.org/doc/man/ets.html) and [DETS](http://www.erlang.org/doc/man/dets.html) are key-value data stores built into [OTP](http://www.erlang.org/doc/). OTP also provides a relational database called [mnesia](http://www.erlang.org/doc/man/mnesia.html) with its own query language called QLC. Both Elixir and Erlang also have a number of libraries for working with a wide range of popular data stores.
+While Phoenix uses the [Ecto project](https://hexdocs.pm/ecto) to interact with the data access layer, there are many other data access options, some even built into the Erlang standard library. [ETS](http://www.erlang.org/doc/man/ets.html) – available in Ecto via [`etso`](https://hexdocs.pm/etso/) – and [DETS](http://www.erlang.org/doc/man/dets.html) are key-value data stores built into [OTP](http://www.erlang.org/doc/). OTP also provides a relational database called [Mnesia](http://www.erlang.org/doc/man/mnesia.html) with its own query language called QLC. Both Elixir and Erlang also have a number of libraries for working with a wide range of popular data stores.
 
 The data world is your oyster, but we won't be covering these options in these guides.
+
+
+[`timestamps/0`]: https://hexdocs.pm/ecto/Ecto.Schema.html#timestamps/0
+[`Ecto.Changeset.cast/3`]: https://hexdocs.pm/ecto/Ecto.Changeset.html#cast/3
+[`cast/3`]: https://hexdocs.pm/ecto/Ecto.Changeset.html#cast/3
+[`Ecto.Changeset.validate_required/3`]: https://hexdocs.pm/ecto/Ecto.Changeset.html#validate_required/3
+[`Ecto.Changeset.validate_format/3`]: https://hexdocs.pm/ecto/Ecto.Changeset.html#validate_format/3
+[`Ecto.Repo`]: https://hexdocs.pm/ecto/Ecto.Repo.html
+[`Repo.insert/1`]: https://hexdocs.pm/ecto/Ecto.Repo.html#c:insert/2
+[`Repo.one/1`]: https://hexdocs.pm/ecto/Ecto.Repo.html#c:one/2
+[`Repo.update/2`]: https://hexdocs.pm/ecto/Ecto.Repo.html#c:update/2
+[`Repo.delete/2`]: https://hexdocs.pm/ecto/Ecto.Repo.html#c:delete/2
+[`Repo.insert_all/3`]: https://hexdocs.pm/ecto/Ecto.Repo.html#c:insert_all/3
+[`Repo.update_all/3`]: https://hexdocs.pm/ecto/Ecto.Repo.html#c:update_all/3
+[`Repo.delete_all/2`]: https://hexdocs.pm/ecto/Ecto.Repo.html#c:delete_all/2
+[`Ecto.Query`]: https://hexdocs.pm/ecto/Ecto.Query.html
+[`from/2`]: https://hexdocs.pm/ecto/Ecto.Query.html#from/2
```

Comment:
Please revert this stule, we don't use it anywhere else in the guides. Plus we should not link ``Ecto.Changeset.cast/3``, just let ex_doc do its job. Thank you.

Reply:
done